zebrad/tests/common/rpc_client.rs

```diff
@@ -2,6 +2,8 @@
 
 use std::net::SocketAddr;
 
+use color_eyre::{eyre::eyre, Result};
```

Comment:
I'll push a commit that fixes these clippy warnings, then approve this PR.

Reply:
I pushed commit 0702797, but I can't approve because I'm the last committer.

Reply:
> I'll push a commit that fixes these clippy warnings, then approve this PR.

Thank you!

> I pushed commit https://github.com/ZcashFoundation/zebra/commit/070279742ef548f3ef2a6e04dfdc1b9e28ff821a, but I can't approve because I'm the last committer.

Hm. I'll redo your commits so you can approve.